backend/django/core/views/api_admin.py

```diff
@@ -48,26 +48,15 @@ def label_distribution(request, project_pk):
         key=lambda label: DataLabel.objects.filter(label=label).count(), reverse=True
     )
 
-    dataset = []
-    # Get first labels
-    for label in labels[0:5]:
-        temp_values = []
-        for u in users:
-            label_count = DataLabel.objects.filter(profile=u, label=label).count()
-            if label_count > 0:
-                temp_values.append({"x": u.__str__(), "y": label_count})
-        if temp_values != []:
-            dataset.append({"key": label.name, "values": temp_values})
-
-    other_values = []
+    user_labels = []
+    
     for u in users:
-        other_count = 0
-        for label in labels[5:]:
-            other_count += DataLabel.objects.filter(profile=u, label=label).count()
-        if other_count > 0:
-            other_values.append({"x": u.__str__(), "y": other_count})
-    if other_values != []:
-        dataset.append({"key": "other labels", "values": other_values})
+        label_count = DataLabel.objects.filter(profile=u, label__in=labels).count()
```

Comment:
I like the updated chart! I do have a note about the backend function. Since we're not splitting the results by label anymore, we can actually just use the query `label_count = DataLabel.objects.filter(profile=u, label__project=project_pk).count()`. This way we can remove all mentions of `labels` entirely which should be a bit faster.

Reply:
fixed in e276187